<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="StackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Menu">
     <property name="styleSheet">
      <string notr="true">QWidget#Menu {
    background: qlineargradient(
        spread: pad,
        x1: 1, y1: 0.045, 
        x2: 1, y2: 1, 
        stop: 0 rgba(255, 120, 0, 255),
        stop: 1 rgba(246, 97, 81, 255)
    );

}

QLabel{
    color: white;
    font-size: 50px;
    font-weight: bold;
    background: transparent;
    padding: 0;
    border: none;
    text-align: center;
}

QGroupBox {
    background-color: #2a2a2a;
    border: 2px solid rgba(255, 120, 0, 255);
    border-radius: 12px;
    padding: 20px;
    margin-bottom: 30px;
}

QPushButton {
    background: qlineargradient(
        spread: pad, 
        x1: 1, y1: 0.045, 
        x2: 1, y2: 1, 
        stop: 0 rgba(255, 120, 0, 255), 
        stop: 1 rgba(246, 97, 81, 255)
    );
    color: white;
    border: none;
    border-radius: 12px;
    font-size: 18px;
    padding: 12px 25px;
    margin: 10px 0;
}

QPushButton:hover {
    border: 1px solid rgba(255, 255, 255, 0.6);
    background: rgba(255, 120, 0, 180);
}

QPushButton:pressed {
    background-color: rgba(255, 120, 0, 220);
    background: rgba(255, 255, 255, 0.2);
}

QPushButton:focus {
    outline: none;
}
</string>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>381</width>
        <height>421</height>
       </rect>
      </property>
      <widget class="QLabel" name="title">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>321</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RaDoTech</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>361</width>
         <height>191</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="createProfileButton">
          <property name="text">
           <string>Create Profile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="CreateProfile">
     <property name="styleSheet">
      <string notr="true">QWidget#CreateProfile{
background: qlineargradient(
        spread: pad,
        x1: 1, y1: 0.045, 
        x2: 1, y2: 1, 
        stop: 0 rgba(255, 120, 0, 255),
        stop: 1 rgba(246, 97, 81, 255)
    );
	color:white;
}

#container{
background-color: #2a2a2a;
    border: 2px solid rgba(255, 120, 0, 255);
    border-radius: 12px;
    padding: 20px;
    margin-bottom: 30px;
}

QPushButton {
    background: qlineargradient(
        spread: pad, 
        x1: 1, y1: 0.045, 
        x2: 1, y2: 1, 
        stop: 0 rgba(255, 120, 0, 255), 
        stop: 1 rgba(246, 97, 81, 255)
    );
    color: white;
    border: 2px solid rgb(46, 46, 46);
    border-radius: 12px;
    font-size: 18px;
    padding: 6px 10px;
}

QPushButton:hover {
    border: 1px solid rgba(255, 255, 255, 0.6);
    background: rgba(255, 120, 0, 180);
}

QPushButton:pressed {
    background-color: rgba(255, 120, 0, 220);
    background: rgba(255, 255, 255, 0.2);
}</string>
     </property>
     <widget class="QGroupBox" name="container">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>511</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color: white;
}

QRadioButton{
color:white;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>470</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Save and Continue</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>471</width>
         <height>441</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="userInfo">
        <item row="6" column="2">
         <widget class="QLineEdit" name="countryText">
          <property name="placeholderText">
           <string>Your country</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="heightLbl">
          <property name="text">
           <string>Height, cm</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lNameLbl">
          <property name="text">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="countryLbl">
          <property name="text">
           <string>Country</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="emailText">
          <property name="placeholderText">
           <string>Email Address</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLineEdit" name="confirmPasswordText">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>12345</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lastNameText">
          <property name="placeholderText">
           <string>Last name</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="birthdayLbl">
          <property name="text">
           <string>Date of Birth</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="firstNameText">
          <property name="placeholderText">
           <string>First name</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="emailLbl">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QDateEdit" name="birthdayEdit">
          <property name="displayFormat">
           <string>MM/dd/yyyy</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="phoneLbl">
          <property name="text">
           <string>Phone Number, +</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="genderLbl">
          <property name="text">
           <string>Gender</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="passwordLbl">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="confirmPasswordLbl">
          <property name="text">
           <string>Confirm password</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <layout class="QHBoxLayout" name="genderOptions">
          <item>
           <widget class="QRadioButton" name="maleButton">
            <property name="text">
             <string>Male</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="femaleButton">
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="otherButton">
            <property name="text">
             <string>Other</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="weightLbl">
          <property name="text">
           <string>Weight, kg</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLineEdit" name="passwordText">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>12345</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="fNameLbl">
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="phoneText">
          <property name="placeholderText">
           <string>12223334444</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="weightBox">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="heightBox">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="Homebtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loginProfile">
     <property name="styleSheet">
      <string notr="true">QWidget#loginProfile {
    background: qlineargradient(
        spread: pad,
        x1: 1, y1: 0.045, 
        x2: 1, y2: 1, 
        stop: 0 rgba(255, 120, 0, 255),
        stop: 1 rgba(246, 97, 81, 255)
    );
    border-radius: 15px;
    padding: 40px 30px;
}

QLabel{
    color: white;
    font-size: 50px;
    font-weight: bold;
    background: transparent;
    padding: 0;
    border: none;
    text-align: center;
}

QGroupBox {
    background-color: #2a2a2a;
    border: 2px solid rgba(255, 120, 0, 255);
    border-radius: 12px;
    padding: 20px;
    margin-bottom: 30px;
}

QPushButton {
    background: qlineargradient(
        spread: pad, 
        x1: 1, y1: 0.045, 
        x2: 1, y2: 1, 
        stop: 0 rgba(255, 120, 0, 255), 
        stop: 1 rgba(246, 97, 81, 255)
    );
    color: white;
    border: 2px solid rgb(46, 46, 46);
    border-radius: 12px;
    font-size: 18px;
    padding: 6px 10px;
}

QPushButton:hover {
    border: 1px solid rgba(255, 255, 255, 0.6);
    background: rgba(255, 120, 0, 180);
}

QPushButton:pressed {
    background-color: rgba(255, 120, 0, 220);
    background: rgba(255, 255, 255, 0.2);
}

QPushButton:focus {
    outline: none;
}
</string>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>381</width>
        <height>421</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>251</width>
         <height>206</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="emailLoginText">
          <property name="placeholderText">
           <string>Login (email)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLoginText">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="enterButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
      <widget class="QLabel" name="title_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>-10</y>
         <width>321</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RaDoTech</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Homebtn_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
